.circleci/config.yml

```diff
@@ -2,16 +2,30 @@ version: 2.1
 jobs:
   test:
     docker:
-      - image: circleci/node:lts
+      - image: dkhamsing/awesome_bot
+    parallelism: 50
     steps:
+      - checkout
       - run:
-          name: Test dependencies
+          name: Test links
           command: |
-            echo "No tests"
-            circleci-agent step halt;
+            cd docs/
+            find . -name '*.md' | circleci tests split > /tmp/test-docs
```

Comment:
this is really cool.

why do you want to allow 403?

Reply:
a bunch of links from etherscan replies with 403 but after that they redirect and are working fine, so it's necessary add that role.

Reply:
Is it possible to allow 403 only for the etherscan domain?

Reply:
@mrice32 good catch! added to witelist the domains who replies 403 but still working